src/shared/components/Animations/ListItem.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import './list-item.less';
+
+export interface ListItemProps {
+  label: string;
+  key: string;
```

Comment:
`key` is a reserved prop I think, so it cannot be here

Reply:
I used id instead